tests/Agent/IntegrationTests/UnboundedIntegrationTests/UnboundedIntegrationTests.csproj

```diff
@@ -38,7 +38,7 @@
   <ItemGroup>
     <PackageReference Include="Common.Logging" Version="3.3.1" />
     <PackageReference Include="Common.Logging.Core" Version="3.3.1" />
-    <PackageReference Include="MySql.Data" Version="8.0.25" />
+    <PackageReference Include="MySql.Data" Version="8.0.30" />
```

Comment:
I think this PackageReference can be removed from the project file for the tests themselves, since I don't think any tests code needs access to methods or types from the MySql client.